crates/stackable-telemetry/src/tracing.rs

```diff
@@ -297,6 +300,7 @@ impl BuilderState for builder_state::Config {}
 #[derive(Default)]
 pub struct TracingBuilder<S: BuilderState> {
     service_name: Option<&'static str>,
+    env_var: Option<&'static str>,
```

Comment:
note: This looks like a left-over from the initial implementation and can be removed.

Reply:
Fixed in 5b6b1b65e6ad83af5c22f3bafce1a2f5ba78ab22